libglusterfs/src/inode.c

```diff
@@ -831,10 +839,22 @@ inode_grep(inode_table_t *table, inode_t *parent, const char *name)
     pthread_mutex_lock(&table->lock);
     {
         dentry = __dentry_grep(table, parent, name, hash);
-        if (dentry) {
+
+        if (!dentry) {
+            pthread_mutex_unlock(&table->lock);
+            gf_msg_callingfn(xl->name, GF_LOG_WARNING, EINVAL,
```

Comment:
Not finding a dentry could be a "normal" issue in some cases, which means that we could flood the logs with this message. I wouldn't use `gf_msg_callingfn()` for this, which is costly. I would also use `GF_LOG_DEBUG` at most.